### DISCUSSION TITLE
Distinct() function not working

### DISCUSSION THREAD

**Discussion by iamwebdev** [strapi/strapi#7245] [Legacy Topics]
How to get distict values from collection. I am having a collection where duplicate cities are storing. I want to get distinct cities from collections.

I am using await strapi.query('property').find().distinct() which is throwing an error that distnint() is not a function

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by phamquocthinh** [comment]
just use
`await strapi.query('property').model.distinct()`
